src/main/java/teammates/ui/controller/InstructorFeedbackPublishAction.java

```diff
@@ -23,21 +22,24 @@ protected ActionResult execute() throws EntityDoesNotExistException {
         FeedbackSessionAttributes session = logic.getFeedbackSession(feedbackSessionName, courseId);
         boolean isCreatorOnly = false;
         
-        new GateKeeper().verifyAccessible(
-                instructor, session, isCreatorOnly, Const.ParamsNames.INSTRUCTOR_PERMISSION_MODIFY_SESSION);
+        new GateKeeper().verifyAccessible(instructor,
+                                          session,
+                                          isCreatorOnly,
+                                          Const.ParamsNames.INSTRUCTOR_PERMISSION_MODIFY_SESSION);
         
         try {
             logic.publishFeedbackSession(feedbackSessionName, courseId);
             
             statusToUser.add(Const.StatusMessages.FEEDBACK_SESSION_PUBLISHED);
             statusToAdmin = "Feedback Session <span class=\"bold\">(" + feedbackSessionName + ")</span> " +
-                    "for Course <span class=\"bold\">[" + courseId + "]</span> published.";
+                            "for Course <span class=\"bold\">[" + courseId + "]</span> published.";
         } catch (InvalidParametersException e) {
             setStatusForException(e);
         }
         
-        nextUrl = nextUrl == null ? Const.ActionURIs.INSTRUCTOR_FEEDBACKS_PAGE : nextUrl;
+        nextUrl = nextUrl == null ? Const.ActionURIs.INSTRUCTOR_FEEDBACKS_PAGE 
```

Comment:
Is there ever a time where this will be null? I ran Eclemma and the null branch is always missed even during the entire suite